Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/angular2App/app/app.module.ts

```diff
@@ -2,12 +2,8 @@ import { HomeModule } from './modules/home/home.module';
 import { AboutModule } from './modules/about/about.module';
 import { SharedModule } from './modules/shared/shared.module';

```

Comment:
@afnpires First of all, thanks a million for your pull request!
When need to import just the required parts of each angular module and not add the angular modules to the vendor. The production builds bundle sizes are smaller then after the production build.

Could you revert this?